packages/vscode-dependencies-validation/test/autofix/eventUtilspec.ts

```diff
@@ -0,0 +1,69 @@
+import type { Uri } from "vscode";
+import * as proxyquire from "proxyquire";
+import { VscodeFileEventConfig } from "../../src/vscodeTypes";
+import { handlePackageJsonEvent } from "../../src/autofix/eventUtil";
+import { configurationProxy, utilProxy } from "../moduleProxies";
+import { SinonMock, createSandbox, SinonSandbox } from "sinon";
+
+describe("eventUtil unit tests", () => {
```

Comment:
See the comment below, I think we need to investigate it in more depth
if it is possible to easily build & **maintain** unit tests for such complex event related flows